.github/workflows/deploy.yml

```
@@ -0,0 +1,71 @@
+    needs: [build-wheels, build-sdist]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v6
+        with:
+          python-version: "3.12"
+      - uses: actions/download-artifact@v7
+        with:
+          path: dist
+          merge-multiple: true
+      - name: Publish package to PyPI
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:27: Double quote to prevent globbing and word splitting

(shellcheck)